WeaponInfo.io/WeaponInfo.io/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Text;
+using System.Xml.Linq;
+
+namespace WeaponInfo.io
+{
+    class program
+    {
+
+        private static void Main()
+        {
+            Console.BackgroundColor = ConsoleColor.White;
+            Console.Clear();
+            Console.ForegroundColor = ConsoleColor.Black;
+            Console.OutputEncoding = Encoding.Unicode;
+            Console.InputEncoding = Encoding.Unicode;
+            Console.Title = "WeaponInfo.io (Докийчук А.И) ";

```

Comment:
Extract it to another method which will be called like "ConsoleConfig". This will make your code more readable

Reply:
I believe in further labs we will have it separately anyway.